src/redist/targets/GenerateLayout.targets

```diff
@@ -16,7 +16,7 @@
 
     <!-- Change these individually to or $(CoreSetupBlobVersion), $(AspNetCoreBlobVersion), or appropriate fixed version depending if corresponding .Ref packages are unpinned. -->
     <NETCoreAppTargetingPackBlobVersion>3.1.0-rtm.19565.2</NETCoreAppTargetingPackBlobVersion>
-    <AspNetCoreTargetingPackBlobVersion>3.1.2-servicing.20068.1</AspNetCoreTargetingPackBlobVersion>
+    <AspNetCoreTargetingPackBlobVersion>3.1.0</AspNetCoreTargetingPackBlobVersion>
```

Comment:
I forget the history here, there is something offputing about this one being unsuffixed, but I think we had to. I don't remember which is considered better. sigh.

Reply:
yeah, i think we want unsuffixed, but had to suffix core-setup for some reason I forget.

Reply:
Unsuffixed only happens manually, I thought? There is no unsuffixed location for 3.1.2, for example. If we want the location that actually corresponds to the 3.1.0 targeting pack, it'd be `3.1.0-rtm.19566.1`